reana_server/rest/info.py

```diff
@@ -217,7 +277,29 @@ def info(user, **kwargs):  # noqa
                 title="Maximum inactivity period in days before automatic closure of interactive sessions",
                 value=REANA_INTERACTIVE_SESSION_MAX_INACTIVITY_PERIOD,
             ),
+            dask_enabled=dict(
+                title="Dask workflows allowed in the cluster",
+                value=bool(DASK_ENABLED),
+            ),
         )
+        if DASK_ENABLED:
+            cluster_information["dask_cluster_default_number_of_workers"] = dict(
+                title="Number of workers in Dask clusters by default",
+                value=REANA_DASK_CLUSTER_DEFAULT_NUMBER_OF_WORKERS,
+            )
+            cluster_information["dask_cluster_max_memory_limit"] = dict(
+                title="Maximum memory limit for Dask clusters",
+                value=REANA_DASK_CLUSTER_MAX_MEMORY_LIMIT,
+            )
+            cluster_information["dask_cluster_default_single_worker_memory"] = dict(
+                title="Memory for one Dask worker by default",
```

Comment:
For the documentation strings (titles), please see the comments to the `reana` and `reana-commons` repositories.